html/projects.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
```

Comment:
I would not put it in the HTML file, perhaps the README of the github repository. This comment is not about a line of code but a general comment about the provenance of a script source and is best in dedicated documentation (i.et the project README) rather than the HTML file.